src/psbt.cpp

```diff
@@ -223,7 +223,7 @@ void UpdatePSBTOutput(const SigningProvider& provider, PartiallySignedTransactio
     // Construct a would-be spend of this output, to update sigdata with.
     // Note that ProduceSignature is used to fill in metadata (not actual signatures),
     // so provider does not need to provide any private keys (it can be a HidingSigningProvider).
-    MutableTransactionSignatureCreator creator(&tx, /* index */ 0, out.nValue, SIGHASH_ALL);
+    MutableTransactionSignatureCreator creator(&tx, /*nInIn=*/0, out.nValue, SIGHASH_ALL);
```

Comment:
Same here. I'm not sure this is an improvement in readability. What is a `nInIn`?

Reply:
`nInIn` is used as comment elsewhere, so this is a pre-existing issue. I think every literal numeric type passed to a function should have an argument.

I can offer to rename `nInIn` here or in a follow-up.

Reply:
Done here in a new commit